package.json

```diff
@@ -25,6 +25,8 @@
     "@mapbox/mapbox-gl-draw": "^1.1.2",
     "@mapbox/mapbox-gl-draw-static-mode": "^1.0.1",
     "@tinymce/tinymce-vue": "^3.2.8",
+    "@trpc/client": "^11.0.0-next-beta.264",
```

Comment:
I recall having issues with diverging versions. Right now, everything else is on `10.43.0` should we standardize?

Reply:
I'll drop back to 10.43 and report back if I have any issues!